apis/repo-manager.pulpproject.org/v1alpha1/pulp_types.go

```diff
@@ -893,8 +893,7 @@ type Cache struct {
 	ExternalCacheSecret string `json:"external_cache_secret,omitempty"`
 
 	// Defines if cache should be enabled.
-	// Default: true
-	// +kubebuilder:default:=true
+	// Default: false
```

Comment:
why?

Reply:
This was causing a bug in cluster restore where it was always deploying redis/cache (even if the "old" cluster hasn't `.cache.enabled` defined in CR)

Reply:
that was the default before, opting out was released with 3.18
https://docs.pulpproject.org/pulpcore/changes.html#id128
https://github.com/pulp/pulpcore/pull/1752